docs/rfcs/002-archival_storage/README.md

```diff
@@ -308,7 +308,7 @@ Request:
 
 ```http
 GET /bags/{spaceId}/{externalId}[?version={version}]
-
+GET /versions[?before={version}]
```

Comment:
Should this be:

`GET /bags/{spaceId}/{externalId}/versions[?before={version}]`

Reply:
Yes.